packages/amplify_datastore_plugin_interface/lib/src/types/models/model_provider.dart

```diff
@@ -12,11 +12,15 @@
  * express or implied. See the License for the specific language governing
  * permissions and limitations under the License.
  */
-
+import 'package:amplify_datastore_plugin_interface/src/types/models/model.dart';
 import 'model_schema.dart';
 
 abstract class ModelProviderInterface {
   String version;
 
   List<ModelSchema> modelSchemas;
+
+  ModelType getModelTypeByModelName(String modelName) {
+    throw UnimplementedError('Your ModelProvider.dart file is outdated. Please upgrade your Amplify CLI and re-run Codegen.');
+  }
```

Comment:
Can you add this to our breaking changes RFC.

Reply:
@Amplifiyer This specific line isn't really a breaking change, but the fact that we are relying on it for Hub certainly is.  I'll add to the RFC.